src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,23 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
+        String score = "";
+        Boolean check = false;
```

Comment:
these both variables are redundant, return score when you've found it. Concerning exception - just throw it after loop finished

Reply:
done